wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Create/WireTextField.swift

```diff
@@ -0,0 +1,146 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import UIKit
+import WireCommonComponents
+import WireDesign
+
+protocol WireTextFieldDelegate: AnyObject {
```

Comment:
More like a proposal than a request for a change, but instead of using delegate we could make good use of Combine here as well by exposing various publishers for the client to subscribe to if he wants to.

Reply:
Wil definitely consider it for a follow up PR. Thanks @jullianm!